styles.css

```diff
@@ -0,0 +1,95 @@
+*{
+    box-sizing: border-box;
+    margin:0;
```

Comment:
super jest ten uniwersalny selektor, często też dodaje się do niego 'padding: 0', ale to już zależy od preferencji developera / zespołu